fixtures/workers-with-assets/public/binding.html

```diff
@@ -0,0 +1 @@
+<h1>✨This is from a user worker binding✨</h1>
```

Comment:
So sorry, this should be a suggestion, but GH doesn't support that? when reviewing per commit :/

`<h1>✨This is from a user Worker binding✨</h1>`